test_integration_app/main.php

```diff
@@ -0,0 +1,108 @@
+            $this->assertEquals(1, $objUtil->isOk($objEvents->create_order()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->flag_content()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->link_session_to_user()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->login()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->logout()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->order_status()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->remove_item_from_cart()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->security_notification()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->transaction()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_account()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_comment()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_listing()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_message()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_post()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_profile()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_content_review()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_order()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->update_password()));
+            $this->assertEquals(1, $objUtil->isOk($objEvents->verification()));
+
+            // Decisions API
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getUserDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getOrderDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getContentDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getSessionDecisions()));
+            $this->assertEquals(1, $objUtil->isOk($objDecisions->getDecisions()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_user()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_order()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_session()));
+            $this->assertEquals(0, $objUtil->isOk($objDecisions->apply_decision_to_content()));
+
+            // Wrokflows API
+            $this->assertEquals(1, $objUtil->isOk($objWorkflows->synchronous_workflows()));
+        
+            // Score API
+            $this->assertEquals(0, $objUtil->isOk($objScore->user_score()));//403
+
+            // Verification API
+            $this->assertEquals(1, $objUtil->isOk($objVerification->Send()));
+            $this->assertEquals(1, $objUtil->isOk($objVerification->resend()));
+            $this->assertEquals(1, $objUtil->isOk($objVerification->check()));
```

Comment:
how this case is working now? You cannot specify the correct code, so you should actually assert that the call is unsuccessful (we discussed this on the call)

Reply:
We are getting apiStatus as "50" for the invalid code. So now I have updated the conditions accordingly.